controllers/comments.controller.js

```diff
@@ -0,0 +1,21 @@
+const { selectCommentsByArticleId } = require("../models/comments.model")
+
+const getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params
+
+  if (isNaN(article_id)) {
```

Comment:
Postgres should throw you an error when the article id isn't a number—see if you can catch it, log it, and then add the error code into your postgresql error handler so that you don't need to repeat this logic in other models!